admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -109,6 +109,105 @@ $product-page-padding-bottom: 80px !default;
   }
 
   // Components
+  .product-header-v2 {
```

Comment:
Do we need to version the header inside a classname? can't we just use product-header?

Reply:
Hmm that's weird I had noted some comments for this but they seem to have disappeared
Ah no it's on the `.product-page-v2` class but it's the same principle for this one

For now we have two different pages at the same time but they share the same CSS so sometimes I need to target only the v2 specifically to avoid breaking the legacy one Ideally we should have created a dedicated new scss file but I guess we didn't because it was more convenient this way

Anyway this will be cleaned later when the legacy page is remove, I won't need this extra scope for v2 page